panels/user-accounts/um-app-permissions.c

```diff
@@ -22,12 +22,16 @@
 #include <flatpak.h>
 #include <gio/gio.h>
 #include <gio/gdesktopappinfo.h>
+#include <glib/gi18n.h>
 #include <strings.h>
 
 #include "gs-content-rating.h"
 
 #include "um-app-permissions.h"
 
+/* The value which we store as an age to indicate that OARS filtering is disabled. */
+static const guint32 oars_disabled_age = (guint32) -1;
```

Comment:
Why not a `#define`?

Reply:
A `static const` allows the compiler to do its own type checking and optimisation. A `#define` is just a string substitution which the compiler doesn’t get to know anything about, which is almost always worse for compilation.